specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3627,8 +3627,8 @@
         "tags": [
           "ThreatIntelligence"
         ],
-        "description": "Create a threat intelligence.",
-        "operationId": "CreateThreatIntelligence",
+        "description": "Create a new threat intelligence indicator.",
+        "operationId": "ThreatIntelligenceIndicator_CreateIndicator",
```

Comment:
Do we need to say Indicator twice?
Why not ThreatIntelligenceIndicator_Create ?

Reply:
ThreatIntelligenceIndicator_CreateIndicator:
In the first part, Indicator is required because TI Connector already took ThreatIntelligence, automatic checks does not allow.
In the second part, CreateIndicator is required this is createIndicator api, validation will give us warning if we dont have that

Reply:
Ah got it. So we're forced to these names,